### PR TITLE
Minor style tweaks

### PR DESCRIPTION
**Pull request by huayunh**
This was originally meant to be part of the React Native accessibility audit. However, I could not figure out a proper way to make a11y work on the storybook. Nonetheless, I checked the showcase and the storybook. Our default style works just fine. There are a couple of color knobs used in the storybook that _might_ be a bit low in the contrast, so I altered those to make it more pxb compliant. 

Changes proposed in this Pull Request:
- Go through storybook stories, and changed any color that might potentially violate the a11y.
- Add a 16px margin for the EmptyState component to have the style match with react & angular
- altered the drawer header subtitle font-weight to 300

### CODE REVIEW

File: components/src/core/drawer/drawer-nav-item.tsx [hunk 1]
```diff
@@ -89,7 +89,7 @@ export const DrawerNavItem: React.FC<DrawerNavItemProps> = (props) => {
                 styles={{
                     root: Object.assign(
                         {
-                            paddingLeft: 16 * (depth > 1 ? depth - 1 : 0),
+                            paddingLeft: 32 * (depth > 1 ? depth - 1 : 0),
```
Comment (joebochill) [inline components/src/core/drawer/drawer-nav-item.tsx:92]: the logic for React looks like this:
image
Should we match that, or is there a compelling reason to bump the spacing to 32 at every level?

File: demos/showcase/ios/Podfile.lock [hunk 2]
```diff
@@ -495,4 +495,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: a15bef56a89b8bc41cd7ceac3ce506774aebb33c
 
-COCOAPODS: 1.8.4
+COCOAPODS: 1.9.3
```
Comment (joebochill) [inline demos/showcase/ios/Podfile.lock:498]: Looks like I need to update 😝

File: demos/storybook/storybook/stories/channel-value.tsx [hunk 3]
```diff
@@ -86,7 +87,7 @@ storiesOf('ChannelValue', module)
                     units={text('units', 'hz')}
                     color="error"
                     theme={{
-                        colors: { error: 'pink' },
+                        colors: { error: red[500] },
```
Comment (joebochill) [inline demos/storybook/storybook/stories/channel-value.tsx:90]: red[500] is the default error color in the theme, so setting it to red[500] manually doesn't actually show 'overriding the theme'. You can pick any other color from our palette that passes accessibility (purple, orange?)

File: components/src/core/drawer/drawer-nav-item.tsx (hunk 1 above)
Comment (huayunh) [reply components/src/core/drawer/drawer-nav-item.tsx:92]: This is matching the react one. The react component is bumping everything up by spacing(4) per depth

Comment (joebochill) [reply components/src/core/drawer/drawer-nav-item.tsx:92]: ah, good catch. I missed the `spacing` part...looks good. 🦀

Comment (huayunh) [reply components/src/core/drawer/drawer-nav-item.tsx:92]: I cannot unplay the Crab Rave song in my head now bc that emoji

**Pull request merged**